src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,44 @@
 import React from 'react';
+import { movieShape } from '../variables';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = (
+  { title, description = '', imgUrl, imdbUrl, imdbId },
+) => (
```

Comment:
ти можеш тут просто написати props, а default окремо прописати, ну це вже як кому зручно

```suggestion
export const MovieCard = props => (
```